packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -86,11 +100,18 @@ class AnnounceSemanticsEvent extends SemanticsEvent {
   /// This property must not be null.
   final TextDirection textDirection;
 
+  /// Determines whether the announcement should interrupt any existing announcement, or queue after it.
```

Comment:
nit: here an elsewhere: please keep doc comment to approx. 80 chars: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters